src/Acquisition/tests.py

```diff
@@ -1663,7 +1704,7 @@ def test_cant_persist_acquisition_wrappers_newstyle():
 
     Check custom pickler:
 
-    >>> from cStringIO import StringIO
+    >>> from io import BytesIO as StringIO
```

Comment:
Better to rename the usage points `BytesIO` (for clarity), than to alias it here.